### PR TITLE
Runtasklistbugfixes

### PR DESCRIPTION
**Pull request by ritikaguptams**
Fix for fastpass tasklist bug

Fix for rename box automation property

bug fixes and feature for delete flyout and save flyout

Fixing task status guid narration

### CODE REVIEW

File: src/App/SaveLoadEditPage.xaml.cs [hunk 1]
```diff
@@ -133,8 +133,70 @@ private async void ConfirmLoad_Click(object sender, RoutedEventArgs e)
...
+            {
+                ConfirmSave.IsEnabled = false;
+                CancelSave.IsEnabled = false;
+            }
+        }
+
+        //Function monitors text box changes to enable/disable the cancel and save buttons for load feature.
+        private void LoadFlyout_TextChanged(Object sender, TextChangedEventArgs e)
+        {
+            if ((!string.IsNullOrWhiteSpace(LoadFlyoutUserPath.Text)) && (!string.IsNullOrWhiteSpace(LoadFlyoutUserPath.Text)))
```
[the remark this answers is not in the document]
Comment (ritikaguptams) [reply src/App/SaveLoadEditPage.xaml.cs:188]: Yes! Thank you for catching that! It is a mistake.

**Review by patelp3989 — APPROVED** [review]

**Pull request merged**